sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -70,7 +70,7 @@
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-core</artifactId>
-      <version>1.49.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
+      <version>1.50.0-beta.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
```

Comment:
Any reason this was changed to the source version of azure-core? If it is needed this will need to fix the dependency tag being used.